### PR TITLE
Show Shared and personal projects as a list

### PR DESCRIPTION
**Pull request by dhaneshnm**
- Add listing logic to ProjectsListView

Fixes BlueBrain/nexus/issues/1458 and BlueBrain/nexus/issues/1459

### CODE REVIEW

**Comment by dhaneshnm** [comment]
Screenshot 2020-09-02 at 16 12 20

**Review by kenjinp — APPROVED** [review]

File: src/subapps/projects/views/ProjectsListView.tsx [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
-        /> */}
+        <div>
+          <h1>Personal Projects</h1>
+          {personalProjects
+            ? personalProjects.map(v => {
+                return (
+                  <div className={'project-container'}>
+                    <ProjectCard
+                      name={v._label}
+                      description={v.description ? v.description : ''}
```
Comment (Stafeeva) [inline src/subapps/projects/views/ProjectsListView.tsx:53]:
```suggestion
                      description={v.description || ''}
```

File: src/subapps/projects/views/ProjectsListView.tsx [hunk 2]
```diff
@@ -1,22 +1,83 @@
...
+        </div>
+        <div>
+          <h1>Shared Projects</h1>
+          {sharedProjects
+            ? sharedProjects.map(v => {
+                return (
+                  <div className={'project-container'}>
+                    <ProjectCard
+                      name={v._label}
+                      description={v.description ? v.description : ''}
```
Comment (Stafeeva) [inline src/subapps/projects/views/ProjectsListView.tsx:71]:
```suggestion
                      description={v.description || ''}
```

File: src/subapps/projects/views/ProjectsListView.tsx [hunk 3]
```diff
@@ -1,22 +1,83 @@
...
+        return v._organizationLabel !== personalOrg;
+      });
+      setSharedProjects(shared);
+    });
+  }, []);
   return (
     <div className="view-container projects-subapp-container">
-      <div>
-        <h1>Projects</h1>
+      <div className={'list-container'}>
```
Comment (Stafeeva) [inline src/subapps/projects/views/ProjectsListView.tsx:43]: curly braces are not necessary here as it is not an interpreted expressions

**Review by Stafeeva — APPROVED** [review]

**Pull request merged**